packages/react/src/components/Notification/stories/ActionableNotification.stories.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { ActionableNotification } from '../../Notification';
+import { action } from '@storybook/addon-actions';
+import mdx from '../Notification.mdx';
+
+// eslint-disable-next-line storybook/csf-component
+export default {
+  title: 'Components/Notifications/Actionable',
+  component: ActionableNotification,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  args: {
+    kind: 'error',
+    lowContrast: false,
+    hideCloseButton: false,
+    ariaLabel: 'closes notification',
+    statusIconDescription: 'notification',
+    onClose: action('onClose'),
+    onCloseButtonClick: action('onCloseButtonClick'),
+  },
+};
+
+export const Actionable = () => (
```

Comment:
Just so this is consistent with the others:

```suggestion
export const Default = () => (
```